[DOCS-12223] updates to network topology map (#32491)
DataDog/documentation: content/en/network_monitoring/devices/device_topology_map.md

## Overview

> The [Network Device Topology Map][2] uses [Cloudcraft][7] diagrams to provide an interactive visual representation of your network's physical connections. The map automatically discovers and displays devices, their interfaces, and the relationships between them. This visualization helps you identify issues in your network devices, understand their upstream and downstream impacts, troubleshoot connectivity problems, and gain insights into how traffic flows through your infrastructure.

Author: Yes great catch!

In the Network Topology Map, the following navigation options are available:

### Group by

1. Under Group By, use **tags** such as `team`, `service`, and `vendor` to select how you want to visualize your devices:

{{< img src="/network_device_monitoring/network_topology_map/device-topology-group_by.png" alt="The navigation option, with view tags for team, service, and vendor selected." style="width:80%;" >}}

### Filter devices

3. Select the **+ Filter** drop down to gain further granular control over what devices are shown on the Device Topology Map.

{{< img src="/network_device_monitoring/network_topology_map/device_topology_filter_2.png" alt="The navigation option, with view filer option selected." style="width:90%;" >}}

**Note:** The **Filter Devices** setting impacts what devices are shown on the Device Topology Map for _all_ queries you might make. For example, if you filter by a device facet in the search bar.

> **Resources dropdown options:**

Reviewer: Feels a little confusing to me to introduce the "Resources dropdown options" idea without describing what this is. We were just talking about a "Filter" dropdown but now I'm confused about the difference between the "Filter" dropdown and the "Resources" dropdown. Per the image, it seems Resources is an item with the Filter dropdown, and this item has its own dropdown? So like a nested dropdown situation?

_NOT_ Unmonitored Device
: Select this option from the **Resources** dropdown to hide devices on the Device Topology Map that are not directly monitored by Network Device Monitoring, but still discovered by LLDP/CDP, and shown on the map from adjacent devices that are monitored by Network Device Monitoring.

Hide _N_ Unconnected Devices

> : Select this option from the **Resources** dropdown to hide any devices that have no link connections. Devices can be unconnected for reasons such as improper configuration, or the device does not support [LLDP/CDP](#troubleshooting).

Reviewer: ```suggestion
: Select this option to hide devices that have no link connections. A device may appear unconnected due to incorrect configuration or because it does not support [LLDP/CDP](#troubleshooting).
```
